src/components/uploader/css/Uploader.css

```diff
@@ -237,14 +237,23 @@ tr.fl-uploader-file-state-uploaded button:before {
  */
 
 /* Fake add files button wrapper ("Browse files") */
+.fl-force-right{
```

Comment:
The "fl-force-right" is defined [here](https://github.com/fluid-project/infusion/blob/master/src/framework/core/css/fluid.css#L8) for the purpose indicated in its name. We should be careful with abusing it for other purposes. If width increase and center text are your intentions, consider defining another css class or even restructuring css classes that are already attached for this div.